docs/ref/class-based-views/generic-display.txt

```diff
@@ -167,8 +167,7 @@ many projects they are typically the most commonly used views.
         </ul>
 
     If you're using pagination, you can adapt the :ref:`example template from
-    the pagination docs <using-paginator-in-view>`. Change instances of
-    ``contacts`` in that example template to ``page_obj``.
+    the pagination docs <using-paginator-in-view>`.
```

Comment:
This should now point to the [_"Paginating a ``ListView``"_](https://docs.djangoproject.com/en/4.0/topics/pagination/#paginating-a-listview).

Reply:
Thanks @felixxm! 👍 I updated the PR